lib/livebook/intellisense/docs.ex

```diff
@@ -185,27 +185,30 @@ defmodule Livebook.Intellisense.Docs do
 
   The function returns the line where the identifier is located.
   """
-  @spec locate_definition(String.t(), definition()) :: {:ok, pos_integer()} | :error
-  def locate_definition(path, identifier)
+  @spec locate_definition(list() | binary(), definition()) :: {:ok, pos_integer()} | :error
+  def locate_definition(path_or_bytecode, identifier)
```

Comment:
We don’t need thee changes right? Because we are no longer passing the byte code?